app/services/assessors/property_assessor.rb

```diff
@@ -50,7 +50,7 @@ def call(submission_date:, properties:, level_of_help:, smod_cap:)
             net_equity:,
             main_home_equity_disregard: equity_disregard.applied,
             smod_allowance: smod_disregard.applied,
-            assessed_equity: equity_disregard.result,
+            assessed_equity: equity_disregard.result.round(2),
```

Comment:
This makes me a little anxious: Why are we adding rounding to this specific figure as part of this ticket?

Reply:
Type casting done in Property model itself so no rounding required here